### PR TITLE
Add AB test metrics eda

### PR DESCRIPTION
**Pull request by mammykins**

### CODE REVIEW

File: notebooks/eda/generating_ab_test_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+    "There is no difference in the proportion of journeys using at least one related link (journey_click_rate) between page variant A and page variant B.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\\begin{equation*}\n",
+    "\\frac{total number of journeys including at least one click on a related link}{total number of journeys}\n",
```
Comment (ff-l) [inline notebooks/eda/generating_ab_test_data.ipynb]: If we're in latex land, use \textrm{} or \text{} or something equivalent, so the wordy bits are not a single word.

Comment (mammykins) [reply notebooks/eda/generating_ab_test_data.ipynb]: thanks I'm new to it and couldn't find a quick way to do it when googling

**Comment by mammykins** [comment]
This is finished, could do with a quick review to confirm I've derived the metrics correctly. 

Could also delete the notebook that doesn't show working as is now redundant @suganya-s given the one I added is succinct version?

We can copy the notebook I added over to our ab_analysis repo. I created a metadata dir in the DATA_DIR to hold the document_type.csv.gz

**Comment by ff-l** [comment]
I think some commits need to be squashed as they are way too brief.

**Pull request closed without merging**